pyxnat/core/pipelines.py

```diff
@@ -2,14 +2,20 @@
 import os
 import json
 import datetime
-import urllib2
-import smtplib
-from copy import deepcopy
-import email.mime.text
-import xml.dom.minidom
 
-import suds.client
-import suds.xsd.doctor
+import six
+if six.PY2:
+    from urllib2 import urlopen
+elif six.PY3:
+    from urllib.request import urlopen
```

Comment:
Where is `urlopen` used? I think is not used at all and we can exclude that import.

Reply:
Actually it is used further down in the commented part. The previous `import urlopen` was failing due to Python 3 incompatibility. Shall we just comment the whole import?

Reply:
Yes, I suggest commenting the import as well.